### ISSUE TITLE
DCOR-Aid throws error on first startup

### ISSUE THREAD

**Issue by B-Hartmann** [DCOR-dev/DCOR-Aid#65]
Windows 10
DCOR-Aid 0.11.10

Fresh install of DCOR-Aid, first startup. The following error is shown and occurs every half a second or second:  
```
Unhandled exception in DCOR-Aid version 0.11.10:
Traceback (most recent call last):
  File "C:\Users\username\AppData\Local\Programs\DCOR-Aid\dcoraid\gui\upload\widget_upload.py", line 407, in update_job_status
    self.setRowCount(len(self.jobs))
  File "C:\Users\username\AppData\Local\Programs\DCOR-Aid\dcoraid\gui\upload\widget_upload.py", line 358, in jobs
    raise ValueError("Job list not initialized!")
ValueError: Job list not initialized!
```

I should mention that I have a virtual environment on that PC with dcoraid installed and I used that package via the command line already before installing the DCOR-Aid app. Maybe some clash in the AppData folder?

**Comment by paulmueller** [comment]
I can reproduce this on Windows.

**Comment by paulmueller** [comment]
This bug is present since version 0.11.8

**Comment by paulmueller** [comment]
Should be fixed with cafe9175a5a5635da44b912c5770b1d89727d136
